spec/models/contact_spec.rb

```diff
@@ -35,6 +35,7 @@
   end
 
   describe '#name' do
+    # FIXME: Mystery guest at work here.
```

Comment:
After reading that [mystery guest](https://robots.thoughtbot.com/mystery-guest) article you linked from another [commit](377a553b7d9c41942c3cd041be63cb53efa27336), I see what you mean here. Thanks.